processor/resourceprocessor/testdata/config.yaml

```diff
@@ -0,0 +1,22 @@
+receivers:
+  examplereceiver:
+
+processors:
+  resource:
+  resource/2:
+    type: "host"
```

Comment:
We have been asking people to add some comments on the test files so we can later point at them as examples of the settings.

Reply:
Done